include/nuttx/fs/fs.h

```diff
@@ -30,7 +30,9 @@
 #include <nuttx/config.h>
 #include <nuttx/compiler.h>
 
+#include <sys/uio.h>
```

Comment:
```suggestion
#include <nuttx/fs/uio.h>
```

Reply:
sys/uio.h is necessary for struct iovec definition. (eg. dev_zero.c needs it.)
i thought you preferred to include headers indirectly like this.